src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyArgs.cs

```diff
@@ -25,10 +25,17 @@ public enum Verification
         /// </summary>
         public IList<Verification> Verifications { get; set; }
 
+#if IS_DESKTOP
         /// <summary>
-        /// Path to the package that has to be signed.
+        /// Path to the package that has to be verified.
         /// </summary>
         public string PackagePath { get; set; }
+#else
+        /// <summary>
+        /// Paths to the packages that has to be verified.
+        /// </summary>
+        public IReadOnlyList<string> PackagePaths { get; set; }
+#endif
```

Comment:
@zivkan Thank you for taking time to suggest this [pattern](https://github.com/NuGet/NuGet.Client/pull/3662#discussion_r488809811). [`nuget.exe`](https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-verify#examples) accepts only one package or a wildcard pattern for verification whereas in [`dotnet`](https://github.com/NuGet/Home/blob/dev/designs/DotNetSignCommands.md#dotnet-nuget-verify) should accept multiple packages for verification. If I add `[Obsolete("Use PackagePaths instead")]` to `VerifyArgs.PackagePath` property build will fail because of existing callers in [`nuget verify code path`](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/Commands/VerifyCommand.cs#L45).

Reply:
> If I add [Obsolete("Use PackagePaths instead")] to VerifyArgs.PackagePath property build will fail because of existing callers in nuget verify code path.

You can change the nuget.exe code to use `PackagePacks = new [] {path};`. It still only accepts 1 path, but uses the new, not-obsolete API

Reply:
Thanks Andy. I did that in my latest commit.